install_files/ansible-base/roles/app/templates/sites-available/focal/journalist.conf

```diff
@@ -0,0 +1,80 @@
+ServerName {{ securedrop_app_apache_listening_address }}
+<VirtualHost {{ securedrop_app_apache_listening_address }}:8080>
+
+# WSGI settings for Flask app for Source Interface
+WSGIDaemonProcess journalist processes=2 threads=30 display-name=%{GROUP} python-path=/var/www/securedrop
+WSGIScriptAlias / /var/www/journalist.wsgi process-group=journalist application-group=journalist
+WSGIPassAuthorization On
+
+# Tell the browser not to cache HTML responses in order to minimize the chance
+# of the inadvertent release or retention of sensitive data. For more, see
+# https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.9.2.
+Header set Cache-Control "no-store"
+
+# Configure X-Sendfile for more efficient large file downloads.
+# Modern versions of WSGI wrapper may make this obsolete, more
+# research required.
+XSendFile        On
+XSendFilePath    /var/lib/securedrop/store/
+XSendFilePath    /var/lib/securedrop/tmp/
+
+Header edit Set-Cookie ^(.*)$ $1;HttpOnly
+Header always append X-Frame-Options: DENY
```

Comment:
As you say below, we might want to prefer "always set" in these cases. See docs here https://httpd.apache.org/docs/current/mod/mod_headers.html#page-header :

> This difference between onsuccess and always is a feature that resulted as a consequence of how httpd internally stores headers for a HTTP response, since it does not offer any "normalized" single list of headers. The main problem that can arise if the following concept is not kept in mind while writing the configuration is that some HTTP responses might end up with the same header duplicated (confusing users or sometimes even HTTP clients). For example, suppose that you have a simple PHP proxy setup with mod_proxy_fcgi and your backend PHP scripts adds the X-Foo: bar header to each HTTP response. As described above, mod_proxy_fcgi uses the always table to store headers, so a configuration like the following ends up in the wrong result, namely having the header duplicated with both values

We'll need to be careful not to set headers that Flask is already setting—to be defensive, those docs recommend:

```
# 'onsuccess' can be omitted since it is the default
Header onsuccess unset X-Foo
Header always set X-Foo "baz"
```

Will do a bit more reading there before changing.